doc/api/tls.md

```diff
@@ -827,16 +827,27 @@ changes:
     pr-url: https://github.com/nodejs/node/pull/26625
     description: Return the minimum cipher version, instead of a fixed string
       (`'TLSv1/SSLv3'`).
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/30637
+    description: Return the IETF cipher name as `standard_name`.
 -->
 
 * Returns: {Object}
-  * `name` {string} The name of the cipher suite.
+  * `name` {string} OpenSSL name for the cipher suite.
+  * `standard_name` {string} IETF name for the cipher suite.
```

Comment:
The snake case in JS is a bit unfortunate, but I guess this module already uses it elsewhere.

Reply:
It seemed consistent to me, but I'm not attached to it. Should I change it?

Reply:
I don't have strong feelings personally since that file already uses a mix of snake case and camelCase.

Reply:
I vote for sticking to camelCase.

Reply:
This is indeed unfortunate... I guess we should try to migrate to camelCase when possible?

Reply:
`standardName` it is.